src/timesync/timesyncd-bus.h

```diff
@@ -0,0 +1,10 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+#pragma once
+
+/***
+  This file is part of systemd.
+***/
```

Comment:
we probably should drop these "This file is part of systemd" headers from the whole tree eventually. But let's at least not add them to new code...